src/common/ui/pages/transferFundsPage.js

```
@@ -0,0 +1,34 @@
+import { BasePage } from './basePage';
+import { expect } from '../../helpers/pwHelpers';
+
+export class TransferFundsPage extends BasePage {
+  constructor(page, userId = 0) {
+    super(page, userId);
+    this.url = 'https://parabank.parasoft.com/parabank/transfer.htm';
+  }
+
+  async open() {
+    await super.open(this.url);
+  }
+
+  async fillTransferForm(fromAccount, toAccount, amount) {
+    await this.step(`Fill transfer form`, async () => {
+      await this.page.fill('input#amount', amount.toString());
+      await this.page.selectOption('select#fromAccountId', fromAccount.toString());
+      await this.page.selectOption('select#toAccountId', toAccount.toString());
```

Similarly, `fromAccount.toString()` and `toAccount.toString()` assume non-null inputs. Normalize both account values before using them (e.g., `String(fromAccount)`), and consider validating they are non-empty so you can fail fast with a helpful error message if data is missing.